sklearn/naive_bayes.py

```diff
@@ -115,6 +115,16 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    fit_prior : boolean
+        Whether to learn class prior probabilities or not.
+        If false, a uniform prior will be used.
+
+    class_prior : array-like, size=(n_classes,)
+        Prior probabilities of the classes. If specified the priors are not
+        adjusted according to the data.
```

Comment:
Regarding the "auto" by default, this is already the case. By default, the class prior is estimated using the data distribution. 

I am not sure if this is what you mean?